documentation/code.md

```diff
@@ -81,8 +82,7 @@ files explaining a package. This hierarchy must include pointers to:
           when there are other possible choices.
 - the main entry points in the code for understanding the
   implementation,
-- resources for learning the libraries and frameworks on which the
-  package depends.
+- resources for learning the libraries and frameworks the package depends on.
```

Comment:
Actually, the original is better. Let's revert this one.

Reply:
Done.